calendar.html

```diff
@@ -0,0 +1,63 @@
+---
+layout: plain
+title: Community Calendar
+description: Meshery is community-built and welcomes collaboration!
+---
+
+
+
+<div style="text-align:center">
+<a href= "https://bit.ly/2SbrRhe"><button class="button"> Add To Your Calendar</button></a>
```

Comment:
@leecalcote people can subscribe using this button here. I know it's not to the best design aspects, but since thats not in the scope of this issue PR, we can plan to enhance it in a different PR.

Reply:
So should I remove the button and keep the bitly link in the file that @leecalcote recently reviewed few minutes ago?

Reply:
Not sure, what say @leecalcote ? I think it's not needed so that this page get enough attention :laughing: